aws/resource_aws_iam_credential_report.go

```diff
@@ -86,7 +86,9 @@ func resourceAwsIamCredentialReport() *schema.Resource {
 }
 
 func resourceAwsIamCredentialReportUpdate(d *schema.ResourceData, meta interface{}) error {
-	d.SetId("iam-credential-report")
+	// RM-2550: (eric-luminal) Fugue sets the ID of this resource as something	// RM-2550: (eric-luminal) Fugue sets the ID of this resource as something
```

Comment:
Something weird going on with the text here

Reply:
thanks, copy paste error.